pluginconfig/pluginconfig-311.yaml

```diff
@@ -177,16 +177,16 @@ versions:
       genevaLogging: osarpint.azurecr.io/acs/mdsd:master.20190228.1
       genevaStatsd: osarpint.azurecr.io/acs/mdm:git-a909a2e76
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
-      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05262020
+      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod07152020
       metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.232
-      azureControllers: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      aroAdmissionController: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      canary: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      etcdBackup: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      metricsBridge: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      startup: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      sync: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      tlsProxy: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
+      azureControllers: osarpint.azurecr.io/openshift-on-azure/azure:v19.0.1
```

Comment:
We never did this Z versioning here. What changed?

Reply:
We've done this once or twice before - happens when only the sync image content changes but none of the plugin code iirc. See:

- https://github.com/openshift/openshift-azure/releases/tag/v10.1.1
- https://github.com/openshift/openshift-azure/releases/tag/v4.1.1

Reply:
I also don't understand why this is being done, but will lgtm anyway.